unabridged-works-volume-1/original/05-babylon-the-great.adoc

```diff
@@ -839,9 +856,15 @@ And this is the wolf in the sheep`'s clothing, which,
 by this fair appearance of the sheep`'s wool on his back,
 covers his ravenous nature from the eyes of the beholders.
 
+====
+
 Now there are three ways of fornication, one of which this spirit is always guilty of,
 sometimes of them all.
 
+[.numbered-group]
```

Comment:
Here begins a numbered group within a numbered group, and there a couple of places where this happens. Idk if I did it right by closing the first group when the second group started, but it seem like it was the only way I could do it. I thought it might be helpful to further indent the group within the group so it isn't confusing to the reader, but maybe you have a better way, or maybe just leave it as is.

Reply:
I don't have a better way, at least yet.  So let's go with this for now, but I made a note to come back and see about supporting nested numbered groups, and I wrote down this chapter as a place where it would be useful.  So, I think if you see more of these, do them the same way for now, but maybe leave me a comment so I can add a note for places to come back and improve at some point.